pkg/dogstatsd/parser_test.go

```diff
@@ -645,7 +642,12 @@ func TestCustomNamespace(t *testing.T) {
 	assert.Equal(t, metrics.HistogramType, parsed.Mtype)
 	assert.Equal(t, 0, len(parsed.Tags))
 	assert.InEpsilon(t, 1.0, parsed.SampleRate, epsilon)
+}
+
+func TestInvalidNamespace(t *testing.T) {
+	parsed, err := parseMetricMessage([]byte("daemon:21|ms"), "testNamespace.")
```

Comment:
Should namespace with a dot at the end be invalid ? I don't see why. Is this agent5 behavior ?